sklearn/metrics/classification.py

```diff
@@ -1833,10 +1833,13 @@ def hamming_loss(y_true, y_pred, labels=None, sample_weight=None):
     y_type, y_true, y_pred = _check_targets(y_true, y_pred)
     check_consistent_length(y_true, y_pred, sample_weight)
 
-    if labels is None:
-        labels = unique_labels(y_true, y_pred)
-    else:
-        labels = np.asarray(labels)
+    if labels is not None:
+        warnings.warn("This parameter is unused.  It was"
+                      " deprecated in version 0.21 and"
+                      " will be removed in version 0.23"
+                      " as the number of labels is identical"
```

Comment:
Please remove `"as the number of labels is identical to y_true.shape[1]"`. It is redundant and does not help users.